pkg/blockstorage/awsebs/awsebs.go

```diff
@@ -206,7 +206,7 @@ func (s *EbsStorage) SnapshotsList(ctx context.Context, tags map[string]string)
 	var fltrs []*ec2.Filter
 	dsi := &ec2.DescribeSnapshotsInput{}
 	for k, v := range tags {
-		fltr := ec2.Filter{Name: &k, Values: []*string{&v}}
+		fltr := ec2.Filter{Name: aws.String("tag:" + k), Values: []*string{&v}}
```

Comment:
Clarified this with Sirish on a call - We modified this because, previously the caller was doing the work of setting up the tags with the additional prefix "tag:" before calling SnapshotList, which is not desirable because we dont want the clients doing provider specific work. 

LGTM